afrr_remuneration/aFRR.py

```diff
@@ -179,6 +183,32 @@ def calc_acceptance_tolerance_band(
         raise ValueError(
             f"MISSING DATA! Not every second has an entry. (Difference = {diffE}). Check input!"
         )
+        
+    # awarded capacity
+    # determine required entries
+    h_1 = setpoint.index[0].ceil("H")
+    h_t = setpoint.index[-1].floor("H")
+    hours = pd.date_range(start=h_1, end=h_t, freq="H")
+    required_product_slices = hours[hours.hour % 4 == 0].shift(-4, freq="H")
```

Comment:
Why is data shift back by 4 hours?

Reply:
The criterion checked considers the awarded capacities of the previous, not the current product time slices